### PR TITLE
[logging] Remove structured log entry

### PR DESCRIPTION
**Pull request by gregnazario**
### Overview
Removes the previous StructuredLogEntry, moves Security logs, tracing, and a config log all to the new logging system!  Figured I'd finish the cleanup of the logs now to remove this mode.

I've been trying to keep naming of certain types consistent, and if they would conflict to have a different name.

### CODE REVIEW

File: consensus/src/epoch_manager.rs [hunk 1]
```diff
@@ -399,10 +399,9 @@ impl EpochManager {
                 .map_err(|err| {
                     error!(
                         SecurityEvent::ConsensusInvalidMessage,
-                        StructuredLogEntry::default()
-                            .data("from_peer", &peer_id)
-                            .data_display("error", &err)
-                            .data("event", &unverified_event)
+                        remote_peer = peer_id,
+                        error = err.to_string(),
```
Comment (bmwill) [inline consensus/src/epoch_manager.rs:403]: I suppose I could have done this before :D We could add some support for the macros to select if you want to do debug, serde, or display based on a sigil

**Review by bmwill — APPROVED** [review]
This is awesome! :D

File: consensus/src/epoch_manager.rs (hunk 1 above)
Comment (gregnazario) [reply consensus/src/epoch_manager.rs:403]: Yeah, I think we can do that in the future, like tracing, I've had to handle this in a couple places.  But, right now just been churning through and getting it done

Comment (bmwill) [reply consensus/src/epoch_manager.rs:403]: 100% agreed

**Pull request closed without merging**